engine/apps/mobile_app/tests/test_your_going_oncall_notification.py

```diff
@@ -47,6 +48,171 @@ def test_shift_starts_within_range(timing_window_lower, timing_window_upper, sec
     )
 
 
+@pytest.mark.django_db
+def test_get_youre_going_oncall_notification_title(make_organization_and_user, make_user, make_schedule):
+    schedule_name = "asdfasdfasdfasdf"
+
+    organization, user = make_organization_and_user()
+    user2 = make_user(organization=organization)
+    schedule = make_schedule(organization, name=schedule_name, schedule_class=OnCallScheduleWeb)
+    shift_pk = "mncvmnvc"
+    user_pk = user.public_primary_key
+    user_locale = "fr_CA"
+    seconds_until_going_oncall = 600
+    humanized_time_until_going_oncall = "10 minutes"
+
+    same_day_shift_start = timezone.datetime(2023, 7, 8, 9, 0, 0)
+    same_day_shift_end = timezone.datetime(2023, 7, 8, 17, 0, 0)
+
+    multiple_day_shift_start = timezone.datetime(2023, 7, 8, 9, 0, 0)
+    multiple_day_shift_end = timezone.datetime(2023, 7, 12, 17, 0, 0)
+
+    same_day_shift = _create_schedule_event(
+        same_day_shift_start,
+        same_day_shift_end,
+        shift_pk,
+        [
+            {
+                "pk": user_pk,
+            },
+        ],
+    )
+
+    multiple_day_shift = _create_schedule_event(
+        multiple_day_shift_start,
+        multiple_day_shift_end,
+        shift_pk,
+        [
+            {
+                "pk": user_pk,
+            },
+        ],
+    )
+
+    maus = MobileAppUserSettings.objects.create(user=user, locale=user_locale)
+    maus_no_locale = MobileAppUserSettings.objects.create(user=user2)
+
+    ##################
+    # same day shift
+    ##################
+    same_day_shift_title = tasks._get_youre_going_oncall_notification_title(
+        schedule, seconds_until_going_oncall, same_day_shift, maus
+    )
+    same_day_shift_no_locale_title = tasks._get_youre_going_oncall_notification_title(
+        schedule, seconds_until_going_oncall, same_day_shift, maus_no_locale
+    )
+
+    assert (
+        same_day_shift_title
+        == f"You're going on call in {humanized_time_until_going_oncall} for schedule {schedule_name}, 09 h 00 - 17 h 00"
+    )
+    assert (
+        same_day_shift_no_locale_title
+        == f"You're going on call in {humanized_time_until_going_oncall} for schedule {schedule_name}, 9:00\u202fAM - 5:00\u202fPM"
+    )
+
+    ##################
+    # multiple day shift
+    ##################
+    multiple_day_shift_title = tasks._get_youre_going_oncall_notification_title(
+        schedule, seconds_until_going_oncall, multiple_day_shift, maus
+    )
+    multiple_day_shift_no_locale_title = tasks._get_youre_going_oncall_notification_title(
+        schedule, seconds_until_going_oncall, multiple_day_shift, maus_no_locale
+    )
+
+    assert (
+        multiple_day_shift_title
+        == f"You're going on call in {humanized_time_until_going_oncall} for schedule {schedule_name}, 2023-07-08 09 h 00 - 2023-07-12 17 h 00"
```

Comment:
`09 h 00` looks a bit weird IMO, maybe it should be `09:00` or at least `09h00`?
I'm not an expert in locales, but I can't see anything similar to this format in the [reference for fr-CA](https://leap.hcldoc.com/help/topic/SSS28S_8.2.1/XFDL_Specification/i_xfdl_r_formats_fr_CA.html).